src/wave.js

```diff
@@ -96,6 +96,8 @@ class Wave extends Component {
       paused,
       children,
       id,
+      svgId,
+      svgPathId,
       d,
```

Comment:
This looks good but we should add these props to the TypeScript definitions.

```tsx
svgId?: string
svgPathId?: string
```

`WaveProps` in https://github.com/woofers/react-wavify/blob/main/src/index.d.ts#L15